src/site/_includes/components/header/header.html

```diff
@@ -0,0 +1,176 @@
+{% from "../button/button.html" import component as button %}
+
+<header class="main-menu">
+  <div class="top">
+    <div class="container main-menu__inner">
+      <div class="logo">
+        <a href="/">
+          <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Kunnen we voor nu niet een `app-icon` component maken dat al die svg's inlaadt als afbeelding? Later kunnen we misschien kijken hoe we dit netter oplossen maar ik vind dit de markup heel bloated maken..

Reply:
App icon is dan gewoon een `img` die met de src naar de icons verwijst, een lege alt heeft en dan gewoon een `name` property mee krijgt van de uiteindelijke icon die die moet tonen.